.travis.yml

```diff
@@ -57,6 +57,12 @@ matrix:
       addons:
         postgresql: '9.6'
 
+    # only run lint on latest Node LTS
+    - node_js: lts/*
+      addons:
+        postgresql: '9.6'
```

Comment:
Probably not necessary?

Reply:
Yes,  possibly. I will remove postgres addon.